wwwroot/inc/popup.php

```diff
@@ -422,20 +421,20 @@ function renderPopupPortSelector()
 	echo '</tr></table>';
 	finishPortlet();
 
-	echo (<<<JSEND
-	<script>
-		$("#filter-obj").autocomplete({
-			source: "?module=ajax&ac=autocomplete&realm=object"
+	addJS (<<<JSEND
```

Comment:
In this code it may make sense to autocomplete only after having 2-3 characters worth of input, otherwise, for example, `LIKE "%a%"` would match lots of irrelevant rows.